pandas/core/strings.py

```diff
@@ -861,6 +861,44 @@ def str_wrap(arr, width, **kwargs):
     return _na_map(lambda s: '\n'.join(tw.wrap(s)), arr)
 
 
+def str_translate(arr, table, deletechars=None):
+    """
+    Map all characters in the string through the given mapping table.
+    Equivalent to standard ``str.translate``. Note that the optional argument
```

Comment:
pls use ":meth:`str.translate`" (and ":func:`string.maketrans`" for another) like other docstring.